packages/api/src/api/helix/video/HelixVideo.ts

```diff
@@ -215,8 +215,12 @@ export class HelixVideo extends DataObject<HelixVideoData> {
 	/**
 	 * The raw data of muted segments of the video.
 	 */
-	get mutedSegmentData(): HelixVideoMutedSegmentData[] {
-		return this[rawDataSymbol].muted_segments.slice();
+	get mutedSegmentData(): HelixVideoMutedSegmentData[] | null {
```

Comment:
This is an unnecessary breaking change. I'd rather return an empty array if Twitch returns null.